GeneralsMD/Code/Libraries/Source/WWVegas/WWLib/lcw.cpp

```diff
@@ -195,6 +195,7 @@ int LCW_Uncomp(void const * source, void * dest, unsigned long )
  *   05/20/1997 JLB : Created.                                                                 * 
  *=============================================================================================*/
 /*ARGSUSED*/
+// unused
```

Comment:
Let's remove this comment. It is also not present in the GEN version of this change.